examples/all_the_things.py

```diff
@@ -15,7 +15,7 @@
 """Example OpenHTF test logic.
 
 Run with (your virtualenv must be activated first):
-python all_the_things.py --config=example_config.yaml
+python all_the_things.py --config-file=example_config.yaml
```

Comment:
The config is actually no longer used in this test, might we well remove the `--config-file` flag entirely here.